services/search/pkg/content/basic.go

```diff
@@ -0,0 +1,30 @@
+package content
+
+import (
+	"context"
+	storageProvider "github.com/cs3org/go-cs3apis/cs3/storage/provider/v1beta1"
+	"github.com/owncloud/ocis/v2/ocis-pkg/log"
+	"time"
+)
+
+type Basic struct {
```

Comment:
I would love to have describing comments for all exported structs and funcs in this PR. But there are a lot and most of them were presumably copied. Could you maybe add comments for the new functions and structs?